app/controllers/publishers/vacancies_controller.rb

```diff
@@ -37,6 +37,7 @@ def review
     end
 
     @vacancy = VacancyPresenter.new(vacancy)
+    @step = :review
```

Comment:
It looks like the `step_current` method takes care of this:

```ruby
def step_current 
  defined?(step) ? step : :review
end
```

So this line can be rewritten:

https://github.com/DFE-Digital/teaching-vacancies/blob/c97d938b16d4bbc441086c00225f8e77caf95aee/app/views/publishers/vacancies/build/_steps.html.slim#L5